src/main/java/io/epirus/console/project/java/JavaTestCLIRunner.java

```diff
@@ -41,7 +41,7 @@
     public void run() {
         try {
             new UnitTestCreator(javaWrapperDir, unitTestOutputDir).generateJava();
-            Console.exitSuccess(
+            System.out.println(
```

Comment:
nit: shouldn't we be using a logger here?

Reply:
We have like 40 instances of sout in the project, so I think we should probably think about how we replace it globally and then fix it in a future PR.

Reply:
We don't seem to be anywhere else, but I agree this would be a good optimization. I'll do a PR sometime soon that should address some more code quality issues.